modules/cnf-topology-aware-lifecycle-operator-installation-cli.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-talo-for-cluster-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="installing-topology-aware-lifecycle-operator-using-cli_{context}"]
+= Installing the Topology Aware Lifecycle Operator by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the Topology Aware Lifecycle Operator (TALO).
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the latest version of {rh-rhacm} Operator.
+* Set up a hub cluster with disconnected registry.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `Namespace` custom resource (CR) for the TALO:
```

Comment:
openshift-operators should be an existing namespace. Please remove this part.

This is similar to how gitops operator is deployed, which is also under openshift-operators namespace.